cts-java/src/test/java/org/ga4gh/cts/api/reads/ReadMethodsEndpointAliveIT.java

```diff
@@ -0,0 +1,179 @@
+    @Test
+    // Normally we'd just pass in the strings here, but the parameters are used
+    // to create final TAP output streams, so the parameter values need to be
+    // legal filenames ... this means, no colons in the readGroupIds!
+    // The workaround is simple enough, and actually seems a bit more readable,
+    // so we'll pass in a (valid) key to a static Map (see readGroupIdMap just below),
+    // and in the test method we'll look up the actual parameter.
+    @Parameters({
+            "TWO_GOOD, NOT_IMPLEMENTED",
+            "ONE_GOOD_ONE_BAD, NOT_IMPLEMENTED",
+            "TWO_BAD, NOT_IMPLEMENTED",
+            "THREE_GOOD, NOT_IMPLEMENTED"
+    })
+    public void multipleReadGroupsNotSupported(String readGroupId, RespCode expStatus) throws Exception {
+        String replacedReadGroupId = readGroupIdMap.get(readGroupId);
+        SearchReadsRequest srr =
+                SearchReadsRequest.newBuilder()
+                                  .setReadGroupIds(Arrays.asList(replacedReadGroupId.split(";")))
+                                  .build();
+        final GAWrapperException e = catchGAWrapperException(() -> client.reads.searchReads(srr));
+        assertThat(e.getHttpStatusCode()).isEqualTo(expStatus.getCode());
+    }
+
+    /**
+     * Map from string acceptable as a file name (for TAP) and the actual
+     * parameters string the test method wants, to get around problem that
+     * GA4GH readGroupID are not necessarily valid filenames
+     */
+    private static Map<String,String> readGroupIdMap;
+    static {
+        readGroupIdMap = new HashMap<>();
+        readGroupIdMap.put("TWO_GOOD", "low-coverage:HG00534;low-coverage:HG00533");
+        readGroupIdMap.put("ONE_GOOD_ONE_BAD", "low-coverage:HG00534;BAD_ID");
+        readGroupIdMap.put("TWO_BAD", "DUMB_ID;BAD_ID");
+        readGroupIdMap.put("THREE_GOOD", "low-coverage:HG00534;low-coverage:HG00533;low-coverage:HG00533");
+    }
+
+    /**
+     * Check that attempting to supply 0 readgroup IDs in a {@link SearchReadsRequest} is not OK.
+     *
+     * @throws Exception the exception
+     */
+    @SuppressWarnings("ThrowableResultOfMethodCallIgnored")
+    @Test
+    public void emptyReadGroupIdIsNotFound() throws Exception {
+        // first get a valid reference to map our read to
+        final String refId = Utils.getValidReferenceId(client);
+
+        final SearchReadsRequest srr =
+                SearchReadsRequest.newBuilder()
+                                  .setReadGroupIds(Collections.emptyList())
+                                  .setStart(0L)
+                                  .setEnd(150L)
+                                  .setReferenceId(refId)
+                                  .build();
+        final GAWrapperException t = catchGAWrapperException(() -> client.reads.searchReads(srr));
+        assertThat(t.getHttpStatusCode()).isEqualTo(HttpURLConnection.HTTP_NOT_IMPLEMENTED);
```

Comment:
I documented it in the schemas documentation branch based on what I thought was consensus regarding the behavior - I guess it wasn't consensus after all!

Please feel free to open an issue against schemas.  We have to document this behavior.